hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/security/ssl/ReloadingX509KeyManager.java

```diff
@@ -176,7 +181,10 @@ private X509ExtendedKeyManager loadKeyManager(CertificateClient caClient)
     }
 
     currentPrivateKey = privateKey;
-    currentCertId = cert.getSerialNumber().toString();
+    currentCertList.clear();
+    for (X509Certificate caCert: newCertList) {
```

Comment:
the name caCert is a little bit misleading here, as the newCertList should contain the service certificate the subordinate CA certs, and the root CA cert, can we just name it c or cert within the for loop?